src/LibraryViewExtension/web/library/library.html

```diff
@@ -115,6 +115,7 @@
         csharpController.on("libraryDataUpdated", function (items) {
             console.log(items);
             refreshLibraryView(libController);
+            csharpController.closeNodeTooltip(true);
```

Comment:
its like the hard way. close the tooltip no matter what.

Reply:
weird, any idea why this is getting called when you place a custom node?

Reply:
it happens only for 1.3 package in 2.0

Reply:
it's strange that it is only custom nodes though, I can't get a ZeroTouch or NodeModel package to pin the tooltip.

Reply:
1.3 package? do you have any custom node packages made in 2.0 to test with? Where would those exist?

Reply:
I don't have any 2.0 package. I tried with LunchBox and Clockwork. But that's only a theory because the nodes are created from `createXMLNodeInstance`. I have to dig more to find that. but this fix is necessary (when there is any exception, then there is a possibility of hanging tool tip).